src/Cake.Core/IO/IDirectory.cs

```diff
@@ -32,7 +32,8 @@ public interface IDirectory : IFileSystemInfo
         /// Deletes the directory.
         /// </summary>
         /// <param name="recursive">Will perform a recursive delete if set to <c>true</c>.</param>
-        void Delete(bool recursive);
+        /// <param name="force">Will delete read-only files if set to <c>true</c></param>
+        void Delete(bool recursive, bool force);
```

Comment:
@DixonDs yes, exactly.
Just wondering if the attribute modification should be done elsewhere? E.g. introduce new interface for that.

Thoughts on this @patriksvensson @devlead